group-coordinator/src/test/java/org/apache/kafka/coordinator/group/assignor/AssignorTestUtils.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.coordinator.group.assignor;
+
+import org.apache.kafka.common.DirectoryId;
+import org.apache.kafka.common.Uuid;
+import org.apache.kafka.image.TopicImage;
+import org.apache.kafka.image.TopicsImage;
+import org.apache.kafka.metadata.LeaderRecoveryState;
+import org.apache.kafka.metadata.PartitionRegistration;
+import org.apache.kafka.server.immutable.ImmutableMap;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AssignorTestUtils {
```

Comment:
I mainly created this class to copy the methods in https://github.com/apache/kafka/blob/trunk/metadata/src/test/java/org/apache/kafka/image/TopicsImageTest.java#L70-L93

I was unable to access the methods in TopicsImageTest while playing around with import-control-group-coordinator.xml. We currently allow `org.apache.kafka.image` package so I'm not sure what is preventing it. Open to suggestions